src/util/events.js

```diff
@@ -57,10 +58,8 @@ export function computeFilteredEvents(events, filters, zipcodes) {
     }
 
     if (filters.zipcode) {
-      const zipcodeLatLng = L.latLng(zipcodes[filters.zipcode]);
-
       const milesFromZipcode = metersToMiles(
-         zipcodeLatLng.distanceTo([event.lat, event.lng])
+        distance([zipcodes[filters.zipcode][1], zipcodes[filters.zipcode][0]], [event.lng, event.lat])
```

Comment:
it seems worth it to me, tho maybe better/easier to change it on both ends in a follow-up pull?